cosmos/operators/local.py

```diff
@@ -390,6 +390,7 @@ class DbtSeedLocalOperator(DbtLocalBaseOperator):
     """
 
     ui_color = "#F58D7E"
+    template_fields: Sequence[str] = ("full_refresh",)
```

Comment:
We need to "inherit" what was defined in the parent class as well:

https://github.com/astronomer/astronomer-cosmos/blob/1bf3a3efd5d4962a29650ea7357afa77018b3018/cosmos/operators/local.py#L93

```suggestion
    template_fields: Sequence[str] = DbtLocalBaseOperator.template_fields  + ("full_refresh",)  # type: ignore[operator]
```

These comments apply to the other operators as well.